src/FitterBase.cc

```diff
@@ -171,7 +172,8 @@ MinimizeResult FitterBase::minimize(std::string const &whatToFit, double nSigmaC
         MeasuredStarList msOutliers;
         FittedStarList fsOutliers;
         int nOutliers = findOutliers(nSigmaCut, msOutliers, fsOutliers);
-        totalOutliers += nOutliers;
+        totalMeasOutliers += msOutliers.size();
+        totalRefOutliers += fsOutliers.size();
```

Comment:
Is it still useful to have `nOutliers` in addition to `msOutliers` and `fsOutliers`?

Reply:
Yes, because it's the number of outliers in that step only (saves doing `msOutliers.size() + fsOutliers.size()` twice). I'll make a comment to that effect.

Reply:
Not sure a comment is necessary, it just occurred to me as a possible side effect.